pkg/cloud/rgraph/rnode/healthcheck/type_trait.go

```diff
@@ -29,7 +29,7 @@ type typeTrait struct {
 	api.BaseTypeTrait[compute.HealthCheck, alpha.HealthCheck, beta.HealthCheck]
 }
 
-func (*typeTrait) FieldTraits(meta.Version) *api.FieldTraits {
+func (*typeTrait) FieldTraits(v meta.Version) *api.FieldTraits {
 	dt := api.NewFieldTraits()
 	// Built-ins
 	dt.OutputOnly(api.Path{}.Pointer().Field("Fingerprint"))
```

Comment:
In `urlmap` it is set to `System`. Which way is correct?

Reply:
I just noticed that this is an error because HC does not support fingerprint